### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception Cannot get children com.intellij.openapi.options.ex.SortedConfigurableGroup@3f635baa java.lang.NullPointerException: Cannot invoke "java.net.URL.toExternalForm()" because "location" is null

### ISSUE THREAD

**Issue by yelkarama** [microsoft/azure-tools-for-java#7311]
IntelliJ build version: 2023.1 EAP IU-231.7515.13
OS: Windows 10
JDK: JetBrains s.r.o. 17.0.6
Plugin version: 3.73.0-2022.3
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,6,6,1921x1161,layout=com.intellij.openapi.wm.impl.IdeRootPane$MyRootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=Cannot get children com.intellij.openapi.options.ex.SortedConfigurableGroup@3f635baa, throwable=java.lang.NullPointerException: Cannot invoke "java.net.URL.toExternalForm()" because "location" is null
	at java.desktop/javax.swing.ImageIcon.<init>(ImageIcon.java:234)
	at com.microsoft.intellij.ui.AzurePanel.$$$setupUI$$$(AzurePanel.java)
	at com.microsoft.intellij.ui.AzurePanel.<init>(AzurePanel.java:66)
	at com.microsoft.intellij.AzureConfigurable.<init>(AzureConfigurable.java:28)
	at com.microsoft.intellij.AzureConfigurableProvider.createConfigurable(AzureConfigurableProvider.java:22)
```

**Comment by Flanker32** [comment]
@yelkarama Thanks a lot for your report and sorry for the trouble. There are some api changes in IntelliJ 2023.1 EAP, we are working on the compatibility jobs for IntelliJ EAP, will update here once there are any updates.

**Comment by raevilman** [comment]
In case it helps...
I fixed my plugin by resetting icon properties in Swing GUI Designer or directly in the XML files.
and then setting the icon programmatically using [AllIcons](https://github.com/JetBrains/intellij-community/blob/idea/231.8109.175/platform/util/ui/src/com/intellij/icons/AllIcons.java) class provided by the SDK. [More info](https://plugins.jetbrains.com/docs/intellij/work-with-icons-and-images.html#platform-vs-custom-icons).

PS: I found this thread while searching for a solution for myself and thought of updating here once mine was fixed. ✌️

**Comment by wangmingliang-ms** [comment]
@raevilman thanks for your suggestion, we fixed this problem in the same way. see 294879079bbd546c7019df7ababc1fe4e2537136

**Comment by wangmingliang-ms** [comment]
close this issue as it has been fixed since v3.74.0. Please upgrade and try again. feel free to reopen this issue if it reappears.
